audit_lambda.py

```diff
@@ -189,7 +189,7 @@ def analyse_pull_request_status(today):
     now = arrow.utcnow()
     pull_requests = storage.read_json(f"{today}/data/activity_prs.json")
     repositories = storage.read_json(f"{today}/data/repositories.json")
-    for repo in repositories.public:
+    for repo in repositories.active:
 
```

Comment:
For style, we should maybe keep consistent with bracket vs dot notation

Reply:
Agreed!

Reply:
I've made those changes.